regression_and_census.R

```diff
@@ -24,36 +184,34 @@ doubleYScale(obj1, obj2, add.ylab2 = TRUE, use.style=FALSE )
 
 CA_PM25HOUR_plot_max <-
   ggplot(PM_LA, aes(week, PM25_wk_avg)) +
+  theme_bw() +
   geom_line(stat="identity")+ 
   ggtitle("Weekly Avg PM 2.5 in LA (AUG - DEC)")+
   labs(y="Level of PM 2.5 (Micrograms/cubic meter (LC))", x = "Date (week, Aug - Dec)")
 
 covid_LA <- ggplot(PM_LA, aes(week, ICU14lag_wkavg)) +
+  theme_bw() +
   geom_line(stat="identity")+ 
   ggtitle("Total number of people currently hospitalized in the ICU in LA")+
   labs(y="Total number of people currently hospitalized in the ICU", x = "Date (AUG - DEC)")
 
 #RAW   
-pdf("NO2_raw_CA_Apr1820.pdf") 
-grid.arrange(CA_PM25HOUR_plot_max, covid_LA, nrow = 2)
-dev.off()
+ggsave("NO2_raw_CA_Apr1820.pdf", arrangeGrob(CA_PM25HOUR_plot_max, covid_LA, nrow = 2),
+       width=10, height=10)
 
 # old code - U N A D J U S T E D   M O D E L S ---------------------------------------
 
 linearmod_unadjust <- lm (y ~ x, data=dataset)
 
-#reading in data and setting working directory
-setwd("/Users/funmiarogbokun/Desktop/CZ Biohub Contractor/Wildfires & Covid Project/Manuscript/Clean_final_datasets/Weekly_data")
-CA_PM25_wk_PM <- read.csv("CA_weekly_lagged_averages.csv", stringsAsFactors = F, header = T)  
-setwd("/Users/funmiarogbokun/Desktop/CZ Biohub Contractor/Wildfires & Covid Project/Census")
-census_counties_2020 <- read.csv("census_CAcountydata_2020.csv")
 
-#cutoff at 30 PM
-PM_CA_below30 <- subset(pol_covid_census_2020, PM25_wk_avg <= 30)
-PM_CA_above30 <- subset(pol_covid_census_2020, PM25_wk_avg > 30)
+ggplot(PM_CA_below30, aes(x=PM25_wk_avg, y=case3lag_wkavg)) +
```

Comment:
This is not super important, but for the sake of consistency, it'd be better to use all ggplot, or all base R graphics.